src/sites.ts

```diff
@@ -28,6 +30,18 @@ export interface searchSite {
     enabled: boolean
 }
 
+export const greynoiseSearch: searchSite = {
+    name: 'GreyNoise',
+    shortName: 'GN',
+    description: 'Provides context and reputation for IP addresses involved in internet scanning.',
+    site: GREYNOISE_SEARCH,
+    ip: true, 
+    hash: true, 
```

Comment:
Please update hash/domain to false since they are not supported by Greynoise

Reply:
ahh thanks for catching. I just made changes